docs/source/faq.rst

```diff
@@ -43,13 +43,13 @@ in tools like ``ps`` and ``top``. This helps for distinguishing the master
 process as well as between masters when running more than one app on a single
 machine. See the proc_name_ setting for more information.
 
-Why is there no HTTP Keep-Alive?
+Why is there no HTTP Keep-Alive header?
 --------------------------------
```

Comment:
```suggestion
Why is there no HTTP Keep-Alive header?
---------------------------------------
```

The adornment needs to match the length of the title, otherwise there's warnings from the document processors.